config/qa-ddb.edn

```diff
@@ -2,4 +2,8 @@
            :requires-auth true}
  :pedestal {:port 8888
             :container-options {:ssl? false}}
- :logger {:level :info}}
+ :logger {:level :warn
```

Comment:
I want QA to log at `warn` so we can focus on warnings and errors. There is *a lot* of noise when logging at `info`